workitem/typegroup/typegroup.go

```diff
@@ -0,0 +1,56 @@
+package typegroup
+
+import (
+	uuid "github.com/satori/go.uuid"
+
+	"github.com/fabric8-services/fabric8-wit/workitem"
+)
+
+// WorkItemTypeGroup represents the node in the group of work item types
+type WorkItemTypeGroup struct {
+	Level                  []int
+	Group                  string
+	Name                   string
+	WorkItemTypeCollection []uuid.UUID
+}
+
+// Use following group constants while defining static groups
+const (
+	GroupPortfolio    = "portfolio"
+	GroupRequirements = "requirements"
+)
+
+// Portfolio0 defines first level of typegroup Portfolio
+var Portfolio0 = WorkItemTypeGroup{
+	Group: GroupPortfolio,
+	Level: []int{0, 0},
```

Comment:
This structure was verified with UI team and this makes their life easy while parsing/using.
For us, this is just static.